src/_modules/experimental/Request.sol

```diff
@@ -204,19 +229,113 @@ library LibRequestClient {
     function put(RequestClient memory self, string memory url) internal pure returns (RequestBuilder memory) {
         return LibRequestBuilder.create(self, Method.PUT, url);
     }
+
+    function setDefaultHeaders(RequestClient memory self, string memory key, string memory value)
+        internal
+        returns (RequestClient memory)
+    {
+        string[] memory values = new string[](1);
+        values[0] = value;
+
+        return setDefaultHeaders(self, key, values);
+    }
+
+    function setDefaultHeaders(RequestClient memory self, string memory key, string[] memory values)
+        internal
+        returns (RequestClient memory)
+    {
+        if (!self.headers.toJsonObject().containsKey(string.concat(".", key))) {
```

Comment:
These seem similar to the stuff in LibHeader, why not reuse those methods?

Reply:
Yeah, not sure why I didn't reuse the headers method. Will check